src/ray/raylet/task_dependency_manager.cc

```diff
@@ -169,7 +214,62 @@ bool TaskDependencyManager::SubscribeDependencies(
   return (task_entry.num_missing_dependencies == 0);
 }
 
-bool TaskDependencyManager::UnsubscribeDependencies(const TaskID &task_id) {
+void TaskDependencyManager::RemoveTaskDependency(const TaskID &task_id,
```

Comment:
this signature doesn't match the one in the `.h` file (the variable name is different)